manticore/platforms/linux.py

```diff
@@ -2334,17 +2456,17 @@ def sys_send(self, sockfd, buf, count, flags):
 
         return count
 
-    def sys_sendfile(self, out_fd, in_fd, offset_p, count):
+    def sys_sendfile(self, out_fd, in_fd, offset_p, count) -> int:
         if offset_p != 0:
-            offset = self.current.read_int(offset_p, self.count.address_bit_size)
+            offset = self.current.read_int(offset_p, self.current.address_bit_size)
```

Comment:
Nice catch!

Reply:
Thanks :)  Adding the type hints there got mypy looking at the implementation, and it complained that `self.count` was nonsense.

There are actually a few dozen places elsewhere in Manticore where there are issues like that, or use-of-undefined variables.  I'm gonna open another PR for those sometime soon.